config/config.go

```diff
@@ -293,6 +294,51 @@ func GetTransport() *http.Transport {
 	return transport
 }
 
+func InitConfig() {
+	// 1) Set up defaults.yaml
+	err := viper.MergeConfig(strings.NewReader(defaultsYaml))
```

Comment:
Neither of these first two `viper.MergeConfig()` calls appear to actually load anything for me. I began by trying to serve a director using a binary named `osdf`. Doing this prints an error that the `Federation.TopologyNamespaceURL` isn't set -- even though it should be by osdf.yaml.

I followed up by adding a print statement right after the first merge statement:
`fmt.Println("Value of Xrootd.ManagerPort: ", viper.GetInt("Xrootd.ManagerPort"))`
This should print `1213` as it appears in defaults.yaml, but it actually prints `0` -- the default int when viper doesn't have another value to print.

As a final sanity check, I tried this print statement right after the first merge:
`fmt.Println("Entire Viper: ", viper.AllSettings())`
this printed:
```
 map[config: debug:true director:map[defaultresponse:] federation:map[discoveryurl:] issuerkey: issuerurl: namespaceurl: origin:map[exportvolume:] server:map[port:8445] stageplugin:map[hook:false mountprefix: originprefix: shadoworiginprefix: token:]]
```
To me this looks like we only have default values defined from the main module's stack of `init` functions.